sim/robots/flood_fill.py

```diff
@@ -655,6 +655,7 @@ def thorough_flood_fill(maze: ExtendedMaze, goals: Set[tuple[int, int]]) -> Robo
         goals,
         explorer=partial(
             flood_fill_thorough_explorer,
-            flood_weight=simple_flood_weight_with_strong_visit_bias,
+            flood_weight=simple_flood_weight,
+            minor_priority=reversed,
```

Comment:
Don't modify this algorithm, we don't need it using known cells when possible and the minor priority is irrelevant